### ISSUE TITLE
'Set Brave as your default browser' dialog on 2nd and nth launch is not translated

### ISSUE THREAD

**Issue by btlechowski** [brave/brave-browser#14177]
Follow up to https://github.com/brave/brave-browser/issues/12203

## Steps to Reproduce

   1. Clean Profile
   1. Run Brave in Polish (PL) locale
   1. Uncheck `Set Brave as your default browser`
   1. Click `OK`
   1. Open New tab
   1. Relaunch browser 

## Actual result:

'Set Brave as your default browser' dialog is not translated
![image](https://user-images.githubusercontent.com/34715963/108064091-78262500-705c-11eb-8008-ac1cefb83188.png)

## Expected result:
'Set Brave as your default browser' dialog is translated

## Reproduces how often: 

Easily reproduced

## Brave version (brave://version info)

Brave | 1.21.56 Chromium: 88.0.4324.152 (Official Build) dev (64-bit)
-- | --
Revision | 6579930fc53b4dc589c042bec9d0a3778326974d-refs/branch-heads/4324@{#2106}
OS | Ubuntu 18.04 LTS

cc @brave/legacy_qa @rebron @mkarolin

**Comment by kjozwiak** [comment]
As per https://github.com/brave/brave-browser/issues/14369#issuecomment-788195579, moving issue into https://github.com/brave/brave-browser/milestone/167 now that https://github.com/brave/brave-core/pull/8093 was merged into `1.22.x` and `1.21.x` via https://github.com/brave/brave-core/pull/8096 & https://github.com/brave/brave-core/pull/8097.

@brave/legacy_qa if it's still an issue, please re-open and remove it from the milestone as there was no code associated with the issues so we can just re-open then and decide if it's release blocking or not.

**Comment by GeetaSarvadnya** [comment]
Verification passed on

```

Brave | 1.21.71 Chromium: 89.0.4389.72 (ಅಧಿಕೃತವಾಗಿ ನಿರ್ಮಿಸಿ) (64-ಬಿಟ್)
-- | --
ಪರಿಷ್ಕರಣೆ | 3f345f156bfd157bd1bea06310e55f3fb2490359-refs/branch-heads/4389@{#1393}
OS | Windows 10 OS Version 2004 (Build 19041.804)

```
- Verified that set brave as default browser dialog is translated in `Kannada`, `Germany` and `France` locales
**Kannada**
![image](https://user-images.githubusercontent.com/38657976/109645716-3aa5b980-7b7d-11eb-9110-2c1e6351975d.png)

**Germany**
Encountered https://github.com/brave/brave-browser/issues/14431
![image](https://user-images.githubusercontent.com/38657976/109646182-de8f6500-7b7d-11eb-9763-72af22ff591d.png)


**France**
![image](https://user-images.githubusercontent.com/38657976/109646048-aee05d00-7b7d-11eb-935a-0ed403b71a9a.png)
